dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import { from, neutral, remSpace } from '@guardian/source-foundations';
+import { ToggleSwitch } from '@guardian/source-react-components-development-kitchen';
+import { useEffect, useState } from 'react';
+
+const cssOverrides = css`
+	padding-bottom: ${remSpace[3]};
+
+	${from.desktop} {
+		padding: ${remSpace[3]} 0;
+		border-top: 1px solid ${neutral[86]};
+	}
+`;
+
+export const FilterKeyEventsToggle = () => {
+	const [checked, setChecked] = useState(false);
```

Comment:
Is there a way to get this value from the CAPI object?

Reply:
I know this is outside this PR but I would expect `CAPI.filterKeyEvents` to be `true` here, no?
<img width="591" alt="Screenshot 2022-02-07 at 17 12 27" src="https://user-images.githubusercontent.com/1336821/152837630-269f9080-5a3c-45db-8c4a-49ea1b57083c.png">

[Example url](https://www.theguardian.com/world/live/2022/feb/07/ukraine-russia-crisis-macron-heads-for-talks-with-putin-while-scholz-and-biden-meet-in-dc-live-coverage.json?dcr&filterKeyEvents=true)

Reply:
That's works nicely on CODE, thanks. I'm not sure why behaviour is sometimes different when running locally but I imagine it's something to do with query params being stripped out somewhere along the way.

Regarding the other point above, I've [made a PR ](https://github.com/guardian/frontend/pull/24615/files)to add the `filterKeyEvents` parameter to the DCR path in `renderJson` so if we decided it's needed that's ready to go.

Reply:
To be honest I don't have a clear understand on the renderArticle vs. renderJson functions but I can say that the 'CAPI' object here (in this context) is the blob that Frontend sends to DCR, and this is what you are passing into `KeyEventsContainer` (via `CAPI.filterKeyEvents`) and now also into `KeyEventsToggle`.

So it sounds like this data is already being sent by Frontend to DCR and perhaps the extra PR isn't needed? But the names of all these function confuses me so I could be misunderstanding.

Reply:
I've updated the Frontend PR to explain a bit more context, we can resolve discussions on there hopefully 🙂